NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -688,12 +688,12 @@ def get_waveforms(self, apply_baseline_correction=None, max_events=1000):
                 selectors=self._select_events)):
 
                 if self._read_calibrated_data:
-                    wfs = wfs
+                    wfs = wfs * units.V
                 else:
                     # wf stores ADC counts
                     if self._convert_to_voltage:
                         # convert adc to voltage
-                        wfs = wfs * (self._adc_ref_voltage_range / (2 ** (self._adc_n_bits) - 1))
+                        wfs = wfs * (self._adc_ref_voltage_range / (2 ** (self._adc_n_bits) - 1)) * units.V
```

Comment:
hm this is not necessary actually because `self._adc_ref_voltage_range` is already defined in volt